cmake/macros/SundialsAddBenchmark.cmake

```diff
@@ -0,0 +1,88 @@
+# Take in an executable, determine how to construct the arguments
```

Comment:
Add copyright statement block to the top of this file (just copy it from another file).